include/lsst/meas/modelfit/Mixture.h

```diff
@@ -347,6 +372,11 @@ class Mixture : public afw::table::io::PersistableFacade<Mixture>, public afw::t
     virtual void write(OutputArchiveHandle & handle) const;
 
 private:
+    template <typename A, typename B, typename C>
+    void evaluateDerivativesImpl(A const & x,
+                                 B & gradient,
+                                 C * hessian,
+                                 bool computeHessian = true) const;
```

Comment:
Might want to start this with an underscores (we aren't universally consistent about whether to do that on all private methods, but we do seem to be doing it in other methods in this class).